jbpm-services/jbpm-kie-services/src/main/resources/META-INF/Servicesorm.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<entity-mappings xmlns="http://java.sun.com/xml/ns/persistence/orm"
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://java.sun.com/xml/ns/persistence/orm orm_2_0.xsd"
-  version="2.0">
+<entity-mappings xmlns=http://xmlns.jcp.org/xml/ns/persistence/orm
+  xmlns:xsi=http://www.w3.org/2001/XMLSchema-instance
+  xsi:schemaLocation=http://xmlns.jcp.org/xml/ns/persistence/orm orm_2_1.xsd
+ version="2.1">
 
```

Comment:
Can you restore the quotes? Without them, I get a validation error on https://www.freeformatter.com/xml-validator-xsd.html